torchinductor/ir.py

```diff
@@ -1481,6 +1481,12 @@ def is_stride_ordered(self, order):
                 return False
         return True
 
+    def is_channels_last_stride_ordered(self):
+        # create channels_last order(NCHW, NCDHW, the C is the first order).
+        order = [0] + list(reversed(range(1, len(self.stride) - 1)))
+        order = [len(order)] + order
```

Comment:
Is the order correct? Regarding a nhwc tensor, the order will be [3, 0, 2, 1]. Is it as expected?

Reply:
Yes, it is expected. For NCHW, the stride order is [3, 2, 1, 0], but for the NHWC, the stride order is [3, 0, 2, 1].